tests/TestUtil/Capture.cpp

```diff
@@ -136,7 +135,7 @@ static void image_stream_callback(k4a_result_t result, k4a_image_t image_handle,
         }
         else
         {
-            fprintf(p_file, "%d, ", (int)image_get_timestamp_usec(image_handle));
+            fprintf(p_file, "%d, ", (int)image_get_device_timestamp_usec(image_handle));
```

Comment:
Same comment here about the cast to (int)

```suggestion
            fprintf(p_file, PRId64 ", ", image_get_device_timestamp_usec(image_handle));
```